components/overview/ReBondModal.js

```diff
@@ -157,6 +160,14 @@ const ReBondModal = withSlideIn(
 					minPossibleStake
 			) {
 				setCalculationDisabled(true);
+			} else if (
+				controllerAccount &&
+				controllerBalances &&
+				transactionFee +
+					apiInstance?.consts.balances.existentialDeposit.toNumber() >
```

Comment:
same as in InvestMore, change this to `transactionFee + (apiInstance?.consts.balances.existentialDeposit.toNumber() / 2)`